fingertip/plugins/os/centos-stream.ks

```diff
@@ -46,13 +49,40 @@ kexec-tools
 %end
 
 %post --erroronfail
+repo --install --name=BaseOS --baseurl=http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/
+repo --install --name=AppStream --baseurl=http://mirror.centos.org/centos/8-stream/AppStream/x86_64/os/
 sed -i 's|\blocalhost\b|{HOSTNAME} localhost|' /etc/hosts
 sed -i 's|^\[main\]$|[main]\nproxy={PROXY}\ndeltarpm=0\nzchunk=0\ninstall_weak_deps=0\ntimeout=900\nmax_parallel_downloads=16|' /etc/dnf/dnf.conf
 sed -i 's|^baseurl=https://|baseurl=http://|' /etc/yum.repos.d/*
 sed -i 's|^metalink=https://|metalink=http://|' /etc/yum.repos.d/*
 sed -i 's|^metalink=\(.*\)$|metalink=\1\&protocol=http|' /etc/yum.repos.d/*
 sed -i 's|gpgcheck=1|gpgcheck=0|' /etc/dnf/dnf.conf
 
+mkdir /etc/yum.repos.d/
```

Comment:
I think repos should be added using `repo --install` command: https://docs.fedoraproject.org/en-US/fedora/rawhide/install-guide/appendixes/Kickstart_Syntax_Reference/#sect-kickstart-commands-repo